src/ResourceManager/Sql/ChangeLog.md

```diff
@@ -24,6 +24,8 @@
 ## Version 4.3.0
 * Fixed issue with importing aliases
 * Get-AzureRmSqlServer, New-AzureRmSqlServer, and Remove-AzureRmSqlServer response now includes FullyQualifiedDomainName property.
+* Add new cmdlet 'Stop-AzureRmSqlElasticPoolActivity' to support canceling the asynchronous operations on elastic pool
```

Comment:
@payiAzure would you mind pulling the latest from the `preview` branch? The content you are adding should be under the `## Current Release` header

Reply:
@cormacpayne pull the latest from preview branch and put my changes under the ##Current Release header